gr-uhd/swig/uhd_swig.i

```diff
@@ -44,6 +44,12 @@
 ////////////////////////////////////////////////////////////////////////
 %ignore gr::uhd::usrp_sink::get_device;
 %ignore gr::uhd::usrp_source::get_device;
+// Also ignore some other UHD types
```

Comment:
These and the other similar RFNoC additions need to be surrounded by `#ifdef ENABLE_UHD_RFNOC` or the equivalent. Otherwise they will be incorporated into the SWIG C++ and then become accessible in Python regardless of whether they are in the gr-uhd library.

Reply:
These are easy to add the `#ifdef` around, since they are in the top-level scope.